### PR TITLE
Made improvements to the build scripts

### PR DESCRIPTION
**Pull request by S0ulf3re**
Per stack overflow question #65991508 I was able to find a fix for why the app wouldn't compile on my machine. I have these proposed changes to fix it. This should now allow the app to compile on any current machine, regardless of architecture

### CODE REVIEW

**Comment by S0ulf3re** [comment]
I think secrets might be important, I just didn't include it because I thought It had sensitive data. Let me try again

**Comment by ggoraa** [comment]
No, do not include them here
they are called secrets for a reason

**Comment by S0ulf3re** [comment]
Me too, Glad I nuked the original repo I had and reforked!
